cognify/run/run.py

```diff
@@ -0,0 +1,54 @@
+import os
+from typing import Optional, Callable
+import logging
+
+from cognify.optimizer.control_param import ControlParameter
+from cognify.optimizer.core import driver
+from cognify.optimizer.plugin import OptimizerSchema
+
+from cognify._signal import _should_exit, _init_exit_gracefully
+
+_init_exit_gracefully(msg="Stopping main", verbose=True)
+
+logger = logging.getLogger(__name__)
+
+def run(
+    *,
+    config_id: str,
+    workflow: str,
+    input: str,
+    control_param: Optional[ControlParameter] = None,
+):
+    if config_id == 'Original':
+        print("Loading original workflow")
+        program = OptimizerSchema.capture(workflow).program
+    else:
+        print(f"Loading workflow with {config_id}")
+        program = load_workflow(config_id=config_id, control_param=control_param)
+    print(f"Running the following input: '{input}'...")
+    result = program(input)
+    print(result)
+
+
+def load_workflow(
```

Comment:
what is load_workflow used for?

Reply:
It's used both internally and can be used by the user in their code if they want to actually use the new workflow in a script. Like they can do `my_qa_workflow = cognify.load_workflow(...)`